web_hosting_manager/app/constants.js

```diff
@@ -17,6 +23,7 @@ const CONSTANTS = {
       scope: null,
       name: pkg.productName,
       vendor: pkg.author.name,
+      customExecPath: isDevMode ? `${cwd}/node_modules/.bin/electron${electronExt} ${cwd}/app` : app.getPath('exe')
```

Comment:
You'll have to provide this as an array as well.

Reply:
I'll have to run more tests on this actually. I found that setting this as an array, `[${cwd}/node_modules/.bin/electron${electronExt} ${cwd}/app]` makes it so that my app is not reachable on the system.

Reply:
That must be because that's incorrect, it should be ```[`${cwd}/node_modules/.bin/electron${electronExt}`,  `${cwd}/app`]```